frontend/src/views/components/UnderlinedWord.tsx

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable no-restricted-imports */
+import { Box, Typography } from "@material-ui/core";
+import React from "react";
+
+interface Props {
+  word: string;
+}
+
+function convertWordToLettersArray(word: string): string[] {
+  const lettersArray: string[] = [];
+
+  for (let i = 0; i < word.length; i++) {
+    lettersArray.push(word[i].toUpperCase());
+  }
+
+  return lettersArray;
+}
+
+export const UnderlinedWord = ({ word }: Props) => {
+  const variant = word.toUpperCase() === "HANGMAN" ? "h1" : "h2";
```

Comment:
Nezinau. Zodi hagnman kaip suprantu naudoji tik pavadinime, tai nebutu geriau ten kur naudoji HANGMAN uzdet variant=h1 o cia nedaryt to const variant? Arba variant paduot per propsus?

Reply:
Padarius variant per propsus gautųsi tas pats kas ir dabar, tik tiek, kad const reikėtų laikyti HANGMAN patį žodį, iš kurio sudarinėtų tą returned elementą, o kita vertus jeigu patį HANGMAN žodžio sudarymą įhardcodinti, tuo atveju man gaunasi jog reikia turėti du kodo gabalus (ar failus), kurie iš esmės daro tą patį. Todėl šiuo atveju paduodant žodį per props'us ir taip nustatant kurį variantą naudoti nereikia perrašinėti to paties kodo du kartus.